### ISSUE TITLE
Cannot make autopep8 and wemake-python-styleguide agree on formatting

### ISSUE THREAD

**Issue by edmondo1984** [wemake-services/wemake-python-styleguide#2298]
### What's wrong

Files formatted via autopep8 trigger a violation to wemake-python-styleguide:

./tests/test_config_factory.py:7:38: WPS318 Found extra indentation
./tests/test_config_factory.py:7:61: WPS319 Found bracket in wrong position

```python

def do_something(
        table_1: pyspark.sql.DataFrame,
        table_2: pyspark.sql.DataFrame,
        config: Config):
```

The offending problem seems to be table_1 tabbing and the last bracket

### How it should be

I should either be able to configure autopep8 to respect those issues, or to fix that automatically in some other way :) 

### Flake8 version and plugins

```json
{
  "dependencies": [],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.9.7",
    "system": "Darwin"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "flake8-bandit",
      "version": "2.1.2"
    },
    {
      "is_local": false,
      "plugin": "flake8-broken-line",
      "version": "0.4.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-bugbear",
      "version": "21.11.29"
    },
    {
      "is_local": false,
      "plugin": "flake8-comprehensions",
      "version": "3.7.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-darglint",
      "version": "1.8.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-debugger",
      "version": "4.0.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-docstrings",
      "version": "1.6.0, pydocstyle: 6.1.1"
    },
    {
      "is_local": false,
      "plugin": "flake8-eradicate",
      "version": "1.2.0"
    },
    {
      "is_local": false,
      "plugin": "flake8-string-format",
      "version": "0.3.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_commas",
      "version": "2.1.0"
    },
    {
      "is_local": false,
      "plugin": "flake8_isort",
      "version": "4.1.1"
    },
    {
      "is_local": false,
      "plugin": "flake8_quotes",
      "version": "3.3.1"
    },
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "naming",
      "version": "0.12.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.8.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.4.0"
    },
    {
      "is_local": false,
      "plugin": "rst-docstrings",
      "version": "0.2.5"
    },
    {
      "is_local": false,
      "plugin": "wemake_python_styleguide",
      "version": "0.16.0"
    }
  ],
  "version": "4.0.1"
}
```

### pip information

pip 21.2.4  via poetry

### OS information

MacOs Big Sur 11.5.2 Chip Apple M1

**Comment by sobolevn** [comment]
Will 

```python
def do_something(
    table_1: pyspark.sql.DataFrame,
    table_2: pyspark.sql.DataFrame,
    config: Config,
):
    ...
```

work? 🤔

**Comment by edmondo1984** [comment]
That would solve one of the two problems, but how do you get autoPEP 8 to format file according to that? I am getting WPS319 WPS318 violations everywhere autopep8  :(

**Comment by sobolevn** [comment]
Sorry, I don't personally use `autopep8` or any other code formatters, but here's what we do:
1. We test that all valid WPS code is valid `autopep8` code: https://github.com/wemake-services/wemake-python-styleguide/blob/master/Makefile#L7
2. I've manually tried to run it on our codebase (it is pretty large), it had nothing to fix

Maybe `autopep8` has some extra options to configure? 🤔

**Comment by edmondo1984** [comment]
@sobolevn thanks, can you share with me your pyproject.toml section ?

```
[tool.autopep8]
max-line-length = 100
in-place = true
aggressive = 4
```

**Comment by sobolevn** [comment]
We don't use any https://github.com/wemake-services/wemake-python-styleguide/blob/master/setup.cfg

**Comment by edmondo1984** [comment]
Do you format the code manually?

**Comment by sobolevn** [comment]
Let's put it the other way: I don't like any tools that change my code 🙂
